pkg/api/serialization_test.go

```diff
@@ -68,13 +70,18 @@ func roundTrip(t *testing.T, codec runtime.Codec, item runtime.Object) {
 		return
 	}
 
+	dataString := string(data)
+	if !strings.HasPrefix(dataString, "{") {
```

Comment:
Sure - but for proto you have this "serialization_proto_test", right? So why changing it here?